can_testbench.py

```diff
@@ -30,27 +28,34 @@
     QComboBox,
     QCheckBox,
     QScrollArea,
+    QLineEdit,
+    QPushButton,
+    QTabBar,
+    QFileDialog,
+    QMessageBox,
+    QGridLayout,
 )
 
-@dataclass
+
+@dataclasses.dataclass
 class DbcSignal:
     """
     A class representing a signal in a CAN message.
 
     Attributes:
     signal (object): The cantools signal object.
-    value (int | float): The value the signal should be (in case of TX) or
+    value (int | float | str): The value the signal should be (in case of TX) or
     is (in case of RX).
     graphValues (deque): A deque of max 100 values that will be graphed.
     Values are the latest values received
-    graph (bool): Whether or not the signal should be graphed.
+    graphed (bool): Whether or not the signal should be graphed.
```

Comment:
Previous naming implies graph is an object instead of a bool, I think there was actually an incorrect existing type hint that treated DbcSignal.graph as an object.